### PR TITLE
Improve functionality on Operator Install to allow for new features

### PR DESCRIPTION
**Pull request by acmenezes**
Solves #97

### CODE REVIEW

File: internal/operator/client.go [hunk 1]
```diff
@@ -27,8 +27,7 @@ type Client interface {
 	DeleteOperatorGroup(ctx context.Context, name string, namespace string) error
 	CreateSubscription(ctx context.Context, data SubscriptionData, namespace string) (*operatorv1alpha1.Subscription, error)
 	DeleteSubscription(ctx context.Context, name string, namespace string) error
-	InstallPlanApprove(namespace string) error
```
Comment (itroyano) [inline internal/operator/client.go:30]: Removing functions from an interface can be scary.   unless these are internal only?

Comment (acmenezes) [reply internal/operator/client.go:30]: They became internal check the installPlan.go file.

Comment (itroyano) [reply internal/operator/client.go:30]: Looked up references to both functions.  looks like they had only one caller each - and you've wrapped that into ApproveInstallPlan.
Just to be sure - nothing outside opcap e.g. in Melvin's UI code calls this directly or anything right?

Comment (itroyano) [reply internal/operator/client.go:30]: https://github.com/search?q=org%3Aopdev+InstallPlanApprove&type=code  

nope.
resolving.

**Review by yashoza19 — APPROVED** [review]
/approve

**Pull request merged**